cmd/rollkit/docs/rollkit_start.md

```diff
@@ -39,6 +39,7 @@ rollkit start [flags]
       --rollkit.da_mempool_ttl uint                     number of DA blocks until transaction is dropped from the mempool
       --rollkit.da_namespace string                     DA namespace to submit blob transactions
       --rollkit.da_start_height uint                    starting DA block height (for syncing)
+      --rollkit.da_submit_options string                DA submit options
```

Comment:
Might be good to document the format (key=value) or some example here.

Reply:
Format is completely DA implementation-dependent, it can be anything that can be parsed by DA.